release/downloadIstioCandidate.sh

```diff
@@ -34,10 +34,21 @@ fi
 NAME="istio-$ISTIO_VERSION"
 URL="https://github.com/istio/istio/releases/download/${ISTIO_VERSION}/istio-${ISTIO_VERSION}-${OSEXT}.tar.gz"
 echo "Downloading $NAME from $URL ..."
-curl -L "$URL" | tar xz
-# TODO: change this so the version is in the tgz/directory name (users trying multiple versions)
-echo "Downloaded into $NAME:"
-ls "$NAME"
-BINDIR="$(cd "$NAME/bin" && pwd)"
-echo "Add $BINDIR to your path; e.g copy paste in your shell and/or ~/.profile:"
-echo "export PATH=\"\$PATH:$BINDIR\""
+curl -L "$URL" | tar xz 
+echo ""
+echo "Istio ${ISTIO_VERSION} Download Complete!"
+echo ""
+echo "Istio has been successfully downloaded into the ${NAME} folder on your system. 🎉\n"
+echo ""
+BINDIR="$(cd "${NAME}/bin" && pwd)"
+echo "Next Steps:\n"
+echo "To add Istio to a Kubernetes cluster, see https://istio.io/docs/setup/kubernetes/#evaluating-istio"
+echo ""
+echo "To configure the istioctl client tool for your workstation,"
+echo "add the ${BINDIR} directory to your environment path variable with:\n"
+echo "\t export PATH=\"\$PATH:${BINDIR}\"\n"  
+echo ""
+echo "Check if your Kubernetes cluster is compatible with Istio by running:"
+echo "\t istioctl verify-install          # verifies Istio installation status"
```

Comment:
This comment is not entirely correct.  It should be something like #pre-check for your cluster and report whether the cluster is ready for Istio installation.

Reply:
@linsun how about: "Begin the Istio pre-installation verification by running:" ?

Reply:
I would just remove this "# verifies Istio installation status"